### PR TITLE
Enhance: EndlessOS

### PR DESCRIPTION
**Pull request by Malix-off**
Adding precision to EndlessOS

### CODE REVIEW

**Comment by Dvlv** [comment]
>  One of the most noob-friendly linux distribution

I _really_ dislike this phrasing.

"Aimed at people new to Linux"

"Familiar to a smartphone or tablet user"

"designed to be very simple"

Something like that would be a lot better.

**Comment by Malix-off** [comment]
Hello @Dvlv !

I agree with you,
let's change that

File: readme.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The Future is Now(tm), try one of these today!
   - [awesome-nix](https://github.com/nix-community/awesome-nix) - A curated list of the best resources in the Nix community
   - Note: The nix community has both an OS and a packaging system and is already large and vibrant, I can't do it justice so just use the list above to dive in
 - [Guix System](https://guix.gnu.org/) - a Linux distribution based on the Guix package manager based on Nix
-- [EndlessOS](https://endlessos.com/home/) - A Debian derivative distribution with a read-only root filesystem managed by OSTree and Flatpak for application delivery and update
+- [EndlessOS](https://endlessos.com/home/) - A Debian derivative distribution with a read-only root filesystem managed by OSTree, and is Flatpak-only for application delivery and update. One of the most noob-friendly linux distribution
```
Comment (Malix-off) [inline readme.md:69]:
```suggestion
- [EndlessOS](https://endlessos.com/home/) - A Debian derivative distribution with a read-only root filesystem managed by OSTree, and is Flatpak-only for application delivery and update. One of the easiest (but constraining) linux distribution
```

Comment (Dvlv) [reply readme.md:69]: To be honest I like that even less.

Comment (Malix-off) [reply readme.md:69]: What's the problem here ?

**Pull request closed without merging**